Makefile

```diff
@@ -0,0 +1,91 @@
+# Makefile to aid with local development and testing
+# This is not required for automated builds
+
+ifeq ($(OS),Windows_NT)
+	PLATFORM := win
+else
+	UNAME := $(shell uname)
+    ifeq ($(UNAME),Linux)
+        PLATFORM := linux
+    endif
+    ifeq ($(UNAME),Darwin)
+        PLATFORM := mac
+    endif
+endif
+
+check-format:
+	cargo fmt -- --check
+
+check-docs:
+	cargo doc --no-deps --workspace --all-features
+
+clippy:
+	cargo clippy --all-features --all-targets -- -D warnings
+
+test-local:
+	cargo test --all-features
+
+test-wasm:
+	cd sdk && wasm-pack test --node
+
+# Full local validation, build and test all features including wasm
+# Run this before pushing a PR to pre-validate
+test: check-format check-docs clippy test-local test-wasm
+	cargo test --all-features
```

Comment:
I think this will run `cargo test --all-features` twice since you're also depending on `test-local`. IMHO this line (34) should be removed.

Reply:
good catch!